lib/SPIRV/libSPIRV/SPIRVNameMapEnum.h

```diff
@@ -613,6 +613,7 @@ template <> inline void SPIRVMap<Capability, std::string>::init() {
   add(CapabilityDebugInfoModuleINTEL, "DebugInfoModuleINTEL");
   add(CapabilitySplitBarrierINTEL, "SplitBarrierINTEL");
   add(CapabilityGroupUniformArithmeticKHR, "GroupUniformArithmeticKHR");
+//   add(CapabilityShaderClockKHR,"ShaderClockKHR");
```

Comment:
Do I understand correctly, that the compilation fails due to undefined `CapabilityShaderClockKHR` here?

Reply:
Why is there a commented statement here?

Reply:
+1 for Dmitry's comment. Looks like this might be the cause for the test fail in CI build.
Please try uncommenting this.

Thanks